src/main/scala/chisel3/util/Bitwise.scala

```diff
@@ -23,14 +33,31 @@ object FillInterleaved {
   def apply(n: Int, in: Seq[Bool]): UInt = Cat(in.map(Fill(n, _)).reverse)
 }
 
-/** Returns the number of bits set (i.e value is 1) in the input signal.
+/** Returns the number of bits set (value is 1 or true) in the input signal.
+  *
+  * @example {{{
+  * PopCount(Seq(true.B, false.B, true.B, true.B))  // dynamically evaluates to 3.U
+  * PopCount(Seq(false.B, false.B, true.B, false.B))  // dynamically evaluates to 1.U
+  *
+  * PopCount("b1011".U)  // dynamically evaluates to 3.U
+  * PopCount("b0010".U)  // dynamically evaluates to 1.U
```

Comment:
Good point. It was meant as a generalization that this could generate dynamic circuitry, but I guess the example below addresses that. Removed unnecessary dynamic references.